Cleaned up code

sethharling opened the pull request:
cleaned the following files:
Update.java
ReadAllOrders.java
ReadOneOrder.java
Main.java (under model)

sethharling committed:
Cleaned up code

cleaned the following files:
Update.java
ReadAllOrders.java
ReadOneOrder.java
Main.java (under model)

snowjtSD reviewed (commented):
Hey, looks good man, I'm sure Jessy may look at this, but the wrapping standard is at 100 char I believe per line and that's the only change I would recommend, the rest is great

S4kRiPp4 commented on app/src/main/java/com/yagni/model/ReadAllOrders.java:
```diff
@@ -3,6 +3,7 @@
 import java.sql.*;
 import javax.swing.*;
 import javax.swing.table.DefaultTableModel;
+
```
delete line 6 whitespace. import com.yagni.controller.*; should be grouped with other imports

S4kRiPp4 commented on app/src/main/java/com/yagni/model/ReadAllOrders.java:
```diff
@@ -16,7 +17,6 @@ public ReadAllOrders(JTable ordersTableIn){
         
         try {
-            
             Statement state = linkDB.getConnection().createStatement(); // creates a statment object
```
move your comment that trails the code at line 20 up, the comment should be on line 20 and the code on line 21. Also, statement is misspelled in the comment.
like this:
line 20 --> // creates a statement object
line 21 -->Statement state = linkDB.getConnection().createStatement();

S4kRiPp4 commented on app/src/main/java/com/yagni/model/ReadOneOrder.java:
```diff
@@ -26,13 +27,12 @@ public void readOneOrder(DbConnection linkDb){
             // creates a DefaultTableModel object
             DefaultTableModel tableModel = (DefaultTableModel) ordersTable.getModel();
 
```
delete whitespace @ line 28. moving the current line 29, up to line 28

S4kRiPp4 commented on app/src/main/java/com/yagni/model/ReadOneOrder.java:
```diff
@@ -47,10 +47,10 @@ public void readOneOrder(DbConnection linkDb){
             String [] row = {orderNum, orderDate, hashEmail, location, productID, productQuantity};
             tableModel.addRow(row);
-            
+
```
delete the whitespace between your catch exception and your closing curly on the while statement.

S4kRiPp4 committed:
Update ReadAllOrders.java

I went ahead and deleted whitespace and move one of your comments up a line for you.

S4kRiPp4 committed:
Update ReadAllOrders.java

S4kRiPp4 committed:
Update ReadAllOrders.java

I fixed the errors I seen in ReadAllOrders.java directly and fixed spelling errors.

S4kRiPp4 committed:
Update ReadOneOrder.java

Fixed spelling/whitespace

S4kRiPp4 committed:
Update TestMain.java

fixed comment

S4kRiPp4 committed:
Update Update.java

Line 71 spell check

S4kRiPp4 committed:
Update ReadOneOrder.java

Spell correction

S4kRiPp4 reviewed (approved):
I made all the changes that I seen w/ my own commits to get these PR's all merged today.
I also pull this PR multiple times locally to confirm the changes and ran a spell check.

I approve and next person to review can go ahead and merge unless they see something critical that needs a change.

KyleZim97 reviewed (approved):
Clean up looks good.

Spoke with Jesse about some of this code and we got it straitened out.

Looks Good.
